AesCtrDecryptor.cpp

```diff
@@ -61,8 +62,13 @@ android::status_t AesCtrDecryptor::decrypt(const android::Vector<uint8_t>& key,
         const SubSample& subSample = subSamples[i];
 
         if (subSample.mNumBytesOfClearData > 0) {
+	  if (secure)
+	    TEE_copy_secure_memory(destination, source,
+				   subSample.mNumBytesOfClearData, offset);
```

Comment:
Is this API defined when USE_AES_TA is not defined?

Reply:
No it isn't good point. Will fix.